R/use_import_from.R

```diff
@@ -0,0 +1,69 @@
+#' Import a function from another package
+#'
+#' `use_import_from()` imports a function from another package by adding the
+#' Roxygen tag `@importFrom` to the package-level documentation (possibly
+#' created with [`use_package_doc()`]). Importing a function from another
+#' package allows you to refer to it without a namespace (e.g., `fun()` instead
+#' of `package::fun()`). Use such imports judiciously, as they make it hard to
+#' tell where a function comes from.
+#'
+#' @param package Name of package to import `fun` from
+#' @param fun Name of function to import
+#' @param load Logical. Re-load with [`pkgload::load_all()`]?
+#'
+#' @return Invisibly, `TRUE` if the package document has changed, `FALSE` if
+#'   not.
+#' @export
+#'
+#' @examples
+#'
+#' \dontrun{
+#' use_import_from("usethis", "ui_todo")
+#' }
+#'
+use_import_from <- function(package, fun, load = is_interactive()) {
+  check_is_package("use_import_from()")
+  check_uses_roxygen("use_import_from()")
+
+  if (!check_has_package_doc()) {
+    return(FALSE)
+  }
+
+  use_dependency(package, "Imports")
+  changed <- roxygen_ns_append(import_from(package, fun)) &&
```

Comment:
`roxygen_update()` doesn't do anything; it just reminds the user. I think it's probably worth fixing the name so would mind renaming `roxygen_update()` to `roxygen_remind()` and then creating an `roxygen_update_ns()` function that does `roxygen2::roxygenise(proj_get(), "namespace")`?